cmd/tailscale/main.go

```diff
@@ -689,9 +702,80 @@ func (s *BackendState) updateExitNodes() {
 			s.Exits = append(s.Exits, exit)
 		}
 	}
+
```

Comment:
Should the sorting happen here, or in ui.go? (I'm not opposed to the logic being here)

Reply:
The current code sorts here, and changing it into ui.go would cause issues with rendering (ask me how I know 😆)